### DISCUSSION TITLE
trouble pushing lightweight changes to gin

### DISCUSSION THREAD

**Discussion by alecristia** [LAAC-LSCP/ChildProject#319] [Q&A] [answered]
I pushed some clips onto zooniverse, which typically just updates a field in a csv. And yet, I've been getting the following for a few days now. There are known problems in our cluster which result in its being unusually slow (since an update was done last Saturday). So perhaps this will be resolved. 

```
(ChildProjectVenv) (base) [acristia@oberon vanuatu-zoo]$ datalad push --to gin
Push to 'gin':  25%|████████████████████████████████████                                                                                                            | 1.00/4.00 [00:00<00:00, 2.66k Steps/s]ssh: connect to host gin.g-node.org port 22: Connection timed out
Transfer data to 'gin':  50%|███████████████████████████████████████████████████████████████████▌                                                                   | 2.00/4.00 [00:00<00:00, 1.72k Steps/sCommandError: 'git -c diff.ignoreSubmodules=none annex copy --batch -z --to gin --fast --json --json-error-messages --json-progress -c annex.dotfiles=true' failed with exitcode 1 under /scratch1/home/acristia/vanuatu-zoo [info keys: stdout_json]
> to gin...
  transfer failed
  This could have failed because --fast is enabled.
git-annex: copy: 1 failed
```

**Comment by lucasgautheron** [comment, marked as answer]
This seems to be due to the network issues with oberon. For instance, I still cannot clone github repositories from oberon (just tried again). The CRI is on it.

**Comment by lucasgautheron** [comment]
Can you try again ? Apparently the network issues were fixed

**Reply by alecristia** [reply]
that worked! so in the future, if people have a similar problem, then perhaps they should check for network issues in their cluster/server
